tests/test_repository_pypi.py

```diff
@@ -61,8 +61,35 @@ def test_generate_hashes_all_platforms(from_line, pypi_repository):
 def test_generate_hashes_without_interfering_with_each_other(
```

Comment:
Probably we should also rename the test name.

Reply:
what about `test_get_file_hash_without_interfering_with_each_other`?

Reply:
yeah, looks good 👍